tox.ini

```diff
@@ -15,27 +15,45 @@ package = skip
 basepython = python3
 setenv =
     LC_ALL = en_US.utf-8
-passenv=
+passenv =
     PYTEST_ADDOPTS
     HYPOTHESIS_PROFILE
+deps =
+    -r{toxinidir}/requirements.txt
+    -r{toxinidir}/test-requirements.txt
```

Comment:
> This gives us conflict requirements where we have pytest!=7.3.2 in test-requirements.txt, but pytest==7.0.1 here.

You are seeing a bug in older versions of pip - how did you hit this? pytest!=7.3.2 and pytest==7.0.1 are not mutually exclusive.

Reply:
> how did you hit this?

By running `tox` on Focal :smile: . It looks like this will install the latest version of pytest rather the one we have pinned.

Reply:
> > how did you hit this?
> 
> By running `tox` on Focal 😄 . It looks like this will install the latest version of pytest rather the one we have pinned.

I mistakenly assumed that we were actually using the pytest version somewhere but it looks like that wasn't in use before so I'll drop it and we can deal with that if/when pytest breaks.